.github/workflows/conda.yml

```diff
@@ -123,11 +123,18 @@ jobs:
       # In Linux/macOS we need the conda-forge channel to install their pinned versions
       - name: Install the Conda package (Linux/macOS)
         if: runner.os != 'Windows'
-        run: conda install --channel conda-forge --channel ./build/conda khiops-core
-      - name: Test that the executables are installed
         run: |
-          MODL -v
-          MODL_Coclustering -v
+          conda install --channel conda-forge --channel ./build/conda khiops-core
+      - name: Add  CONDA_PREFIX to shared PATH
+        run: |
+          echo "$CONDA_PREFIX/bin" >> $GITHUB_PATH
```

Comment:
Why not `echo "CONDA_PREFIX=$CONDA_PREFIX" >> $GITHUB_ENV`? This seems cleaner to me, as `CONDA_PREFIX` would then just be in the environment, as in real conditions, right?